oxide/lib.go

```diff
@@ -22,75 +22,98 @@ const TokenEnvVar = "OXIDE_TOKEN"
 // HostEnvVar is the environment variable that contains the host.
 const HostEnvVar = "OXIDE_HOST"
 
+// Config is the configuration that can be set on a Client.
+type Config struct {
+	// Base URL of the Oxide API including the scheme. For example,
+	// https://api.oxide.computer.
+	Address string
```

Comment:
I just realised that since we call the environment variable `OXIDE_HOST` this should probably be `Host` as well
```suggestion
	Host string
```